src/config.py

```diff
@@ -453,6 +453,18 @@ def log_dirs(self) -> str:
             [os.fspath(storage.location) for storage in self.charm.model.storages["data"]]
         )
 
+    @property
+    def inter_broker_protocol_version(self) -> str:
+        """Creates the protocol version from the kafka version.
+
+        Returns:
+            string with the `major.minor` version
+        """
+        # Remove patch number from full vervion.
+        major_minor = self.charm.upgrade.current_version.split(".", maxsplit=2)
+        major_minor.pop()
+        return ".".join(major_minor)
```

Comment:
**nitpick:** Could we use indexing here? 

```python
major_minor = self.charm.upgrade.current_version.split(".")
return ".".join(major_minor[:-1])
```

Reply:
I believe it safer the one proposed by Raul as we should rather stick to major.minor, if we ever had a 3.2.1.1, using the other would result in a major.minor.patch version, which we don't want

Reply:
Indexing is not so clear directly on the string, because version might be `10.11.2` one day, so [:-1] or [:2] would not work at some point

Reply:
`major_minor` is a list though right?

Reply:
Ah yes sorry. I was thinking about using indexing directly on the string.